webapp/lib/pubsub.dart

```diff
@@ -20,8 +20,8 @@ class PubSubClient extends DocPubSubUpdate {
 
   PubSubClient(this.publishUrl, this.user);
 
-  /// Publish the specified exception.
-  /// Callers should catch and handle IOException.
```

Comment:
`PubSubException` is only one of the exceptions that can be thrown here.
Perhaps `Callers should catch and handle PubSubException and IOException` instead?

Reply:
Good point, thanks. But I don't think IO exception exists for web code? IIRC, this comment was left out of date from when we accidentally imported `dart:io` and had `PubSubException` inherit from `IOException`

Looking at the `http` library, it looks like the only exception they define is `ClientException`, so I've added that instead. If there's another exception that would be good to list here, please let me know.

Reply:
@marianamarasoiu Ah yes. That sounds right. Thanks!